lib/modulr/api/customers_service.rb

```diff
@@ -1,12 +1,40 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+
 module Modulr
   module API
     class CustomersService < Service
       def find(id:)
         response = client.get("/customers/#{id}")
         Resources::Customers::Customer.new(response.env[:raw_body], response.body)
       end
+
+      def create(type:, legal_entity:, **opts)
+        payload = {
+          type: type,
+          legalEntity: legal_entity,
+        }
+
+        payload[:externalReference] = opts[:external_reference] if opts[:external_reference]
+        payload[:name] = opts[:name] if opts[:name]
```

Comment:
Some fields are mandatory or conditioanl mandatory. Name is one of them so we should control it from client? 🤔

Reply:
Yep, but after discussing, we are more on the side of reducing the amount of logic in this gem, the way that @Nachooc5 comments here: https://github.com/devengoapp/modulr-ruby/pull/15#discussion_r1162480348.

So introducing that logic would be a step in the opposite direction, would make the gem more dependant of Modulr.

Of course, open to discussion as usual.